watertap/unit_models/zero_order/ion_exchange_zo.py

```diff
@@ -41,9 +41,10 @@ def build(self):
         pump_electricity(self, self._Q)
 
         # mutable parameter; default value found in WT3 for anion exchange
-        self.eta_pump.set_value(0.8)
-        # mutable parameter; default value of 2 bar converted to feet head
-        self.lift_height.set_value(69.91052 * pyunits.feet)
+        if self.config.process_subtype != "clinoptilolite":
```

Comment:
A minor point, but for future maintenance, it might be better to call out the exceptions instead:

```
if self.config.process_subtype != "clinoptilolite":
    pass
else:
    self.eta_pump.set_value(0.8)
    # mutable parameter; default value of 2 bar converted to feet head
    self.lift_height.set_value(69.91052 * pyunits.feet)
```

Reply:
We could do that, except the logic would be the other way around. Basically, the default WT3 version specified said parameter values, while for this "clinoptilolite" subtype, I want to ignore those specifications. So, for now, I only check whether subtype is or isn't "clinoptilolite". If it isn't, keep the old specifications from WT3; if it is "clinoptilolite", don't touch the default parameter values which are typically used across most other models with pump electricity.

Reply:
Sorry, that is what I meant.